py4DSTEM/process/diffraction/crystal.py

```diff
@@ -118,6 +133,77 @@ def from_pymatgen_structure(structure, conventional_standard_structure=True):
         numbers = np.array([s.species.elements[0].Z for s in structure])
 
         return Crystal(positions, numbers, cell)
+    
+    def create_unitcell_from_params(latt_params, 
+                                   elements, 
+                                   positions, 
+                                   space_group = None, 
+                                   lattice_type = 'cubic', 
+                                   from_cartesian = False):
+        
+        '''
+        Generates pymatgen unit cell manually from user inputs
+    
+        Accepts:
+                latt_params:         (list) list of lattice parameters for example for cubic: latt_params = [a]
+                                     for hexagonal: latt_params = [a, c], for monoclinic: latt_params = [a,b,c,beta]
+                                     and in general: latt_params = [a,b,c,alpha,beta,gamma]
+                elements:            (list) list of elements, for example for SnS: elements = ["Sn", "S"]
+                positions:           (list) list of (x,y,z) positions for each element present in the elements, default: fractional coord
+                space_group:         (optional) (string) space group of the crystal system, if specified, unit cell will be created using 
+                                     pymatgen Structure.from_spacegroup function
+                lattice_type:        (string) type of crystal family: cubic, hexagonal, triclinic etc; default: 'cubic'
+                from_cartesian:      (bool) if True, positions will be considered as cartesian, default: False
+                write_xyz:           (bool) if True, write xyz for visualization in Vesta, default: False
```

Comment:
This argument in the docstring doesn't correspond to any in the function signature.